src/pages/Calendar/Caption/caption.style.ts

```diff
@@ -0,0 +1,82 @@
+import styled from "styled-components/native";
+import { ColorScheme } from "@styles/globalStyles";
+import { FontScheme } from "@styles/globalStyles";
+
+export const Sc = {
+  Captions: styled.View`
+    flex-direction: row;
+    gap: 96px;
+  `,
+
+  LeftCaptions: styled.View`
+    flex-direction: column;
+  `,
+
+  RightCaptions: styled.View`
+    flex-direction: column;
+  `,
+
+  GroupCaptions: styled.View`
+    flex-direction: row;
+    margin-bottom: 13px;
```

Comment:
Also just a challenge for you, as it doesn't really matter in this case.

Make it so the `margin-bottom: 13px` is only applied on the first item of the group so for example, below only the first GroupCaptions should have this margin, the one below it should have margin-bottom set to 0

```js
<Sc.Division>
          <Sc.GroupCaptions>
            <Sc.PinkCircle />
            <Sc.Text>1° dia do ciclo</Sc.Text>
          </Sc.GroupCaptions>

          <Sc.GroupCaptions>
            <Sc.DottedPurpleCircle />
            <Sc.Text>Ciclo Menstrual</Sc.Text>
          </Sc.GroupCaptions>
        </Sc.Division>
```